recipes/riskmatrix/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "riskmatrix" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99c460997581cf8c083c57b13fa825e44ab3d0633e7911ac05f37ff3d4ac20fb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```

Comment:
Please remove the `skip` line, add `noarch: python` instead, and in the `host` and `run` section use `python >=3.7`, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages.